### PR TITLE
Add ColorVolume, rename CanvasColorMetadata, remove Canvas Float

### PR DESCRIPTION
**Pull request by ccameron-chromium**

### CODE REVIEW

**Comment by palemieux** [comment]
See #92

File: hdr_html_canvas_element.md [hunk 1]
```diff
@@ -39,40 +39,44 @@ Add a new attribute to `ScreenAdvanced` to indicate the HDR headroom currently a
   }
 ```
 
-The high dynamic range headroom of a display is computed as:
+See the below discussion of the display of floating point color values for more details.
```
Comment (kenrussell) [inline hdr_html_canvas_element.md:42]: Perhaps: "See below for more details of the display of floating point color values"?

**Review by kenrussell — APPROVED** [review]
Looks good to me overall. One small comment.

**Review by palemieux — APPROVED** [review]
I would specify the units of minimumLuminance/maximumLuminance

File: hdr_html_canvas_element.md (hunk 1 above)
Comment (ccameron-chromium) [reply hdr_html_canvas_element.md]: Good idea -- updated.

**Comment by ccameron-chromium** [comment]
> I would specify the units of minimumLuminance/maximumLuminance

Changed them to I would specify the units of minimumLuminanceNits and maximumLuminanceNits.

**Pull request merged**